### DISCUSSION TITLE
Simple Commerce Default Order Template

### DISCUSSION THREAD

**Discussion by CodePranjal** [duncanmcclean/simple-commerce#1155] [Help]
Hiya, I am trying to find Simple Commerce default template for order_placed emails. I see CP and EMAIL folder in git hub online
but its not in he plugin we've installed when we purchase simple commerce. do i need to create it?

when i did "php artisan make:notification OrderPaidNotification" as per documentation it gives me email.blade file. which doesno't match with the default one i am getting in HELO.

when I place order I am getting email using HELO with a default template with purchase information. but I could not find it in simple commerce. may be you could guide where to look? 

i am on the latest version of statamic (5.25.0) and simple commerce (7.4.7).

Thanks.

**Comment by duncanmcclean** [comment]
Does this help? https://simple-commerce.duncanmcclean.com/notifications#content-custom-notifications

**Reply by CodePranjal** [reply]
I've tried that, it gives me file -> email.blade.php inside vendor. 
but in git you have a email folder. I am bit confused

**Reply by duncanmcclean** [reply]
What are you wanting to customise? Just the content of the email?

If so, you should be able to copy [this file](https://github.com/duncanmcclean/simple-commerce/blob/7.x/resources/views/emails/customer_order_paid.blade.php) (or one of the other email templates in the same directory) into your app's `resources/views/vendor/simple-commerce/emails` directory with the same filename as it appears on GitHub.

**Reply by CodePranjal** [reply]
yes just the content for email. ok so i am gonna create it as the git-hub. alright. thanks
